src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Ball ball = new Ball();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
+        return ball;
+    }
```

Comment:
three points:

1. you don't need this `ball` field, `getRandomBall()` method should return "brand-new" balls
2. you should better define a `Random` field in this class, so that you don't create a new instance of it every time this method is called
3. `100` is a magic number - you should create a dedicated contants for specifying the max ball number

```suggestion
    private static final int MAX_BALL_NUMBER = 100;
    private final ColorSupplier colorSupplier = new ColorSupplier();
    private final Random random = new Random();

    public Ball getRandomBall() {
        Ball ball = new Ball();
        ball.setColor(colorSupplier.getRandomColor());
        ball.setNumber(randim.nextInt(MAX_BALL_NUMBER ));
        return ball;
    }
```